src/main/java/main.java

```diff
@@ -0,0 +1,137 @@
+        SignUpInteractor signUpInteractor = new SignUpInteractor(signUpDatabaseAccess, signUpPresenter);
+        SignUpController signUpController = new SignUpController(signUpInteractor);
+        SignUpView signUpView = new SignUpView(signUpController);
+
+        // initialize stuff for make_comment
+        MakeCommentViewModel makeCommentViewModel = new MakeCommentViewModel();
+        MakeCommentPresenter makeCommentPresenter = new MakeCommentPresenter(makeCommentViewModel);
+        MakeCommentDatabaseAccess makeCommentDatabaseAccess = new MakeCommentDatabaseAccess(numCommentsCreatedFilePath, commentsFilePath);
+        MakeCommentInteractor makeCommentInteractor = new MakeCommentInteractor(makeCommentDatabaseAccess, makeCommentPresenter);
+        MakeCommentController makeCommentController = new MakeCommentController(makeCommentInteractor);
+        MakeCommentView makeCommentView = new MakeCommentView(makeCommentController);
+
+        // initialize stuff for timer
+        PostFactory postFactory = new PostFactory();
+        TimerDataAccess timerDataAccess = new TimerDataAccess(postsFilePath, postFactory);
+        TimerInteractor timerInteractor = new TimerInteractor(timerDataAccess, deletePostInteractor);
+        TimerController timerController = new TimerController(timerInteractor);
+
+        // initialize stuff for log in
+        LogInViewModel logInViewModel = new LogInViewModel();
+        LogInPresenter logInPresenter = new LogInPresenter(logInViewModel);
+        LogInDatabaseAccess logInDatabaseAccess = new LogInDatabaseAccess(usersFilePath);
+        LogInInteractor logInInteractor = new LogInInteractor(logInDatabaseAccess, logInPresenter);
+        LogInController logInController = new LogInController(logInInteractor);
+        LogInView logInView = new LogInView(logInController);
+
+        // initialize stuff for log out
+        LogOutViewModel logOutViewModel = new LogOutViewModel();
+        LogOutPresenter logOutPresenter = new LogOutPresenter(logOutViewModel);
+        LogOutInteractor logOutInteractor = new LogOutInteractor(logOutPresenter);
+        LogOutController logOutController = new LogOutController(logOutInteractor);
+        LogOutView logOutView = new LogOutView(logOutController);
+
+        // initialize stuff for favourite_post
+        UserFactory userFactory = new UserFactory();
+        FavourtiteViewModel favourtiteViewModel = new FavouriteViewModel();
+        FavouritePresenter favouritePresenter = new FavouritePresenter(favourtiteViewModel);
+        DataAccess dataAccess = new DataAccess(postFactory, userFactory, generalPath);
+        FavouriteInteractor favouriteInteractor = new FavouriteInteractor(dataAccess, favouritePresenter);
+        FavouriteController favouriteController = new FavouriteController(favouriteInteractor);
+        FavouriteView favouriteView = new FavouriteView(favouriteController);
```

Comment:
Please remove this code as the view will be initialized in the main view with every refresh